### DISCUSSION TITLE
Uso de npm run preview

### DISCUSSION THREAD

**Discussion by juanmadev5** [ArisGuimera/JetpackComposePro#43] [Q&A]
No sé si solo me pasa a mi, pero cuando ejecuto el 'npm run dev' puedo ver perfectamente los cambios que se hizo en la pagina, pero cuando ejecuto 'npm run preview' me sale 404 not found y path:/
Hay algo que esté haciendo mal?

**Comment by thonyguev** [comment]
Primero generas la build con `npm run build` y luego la preview `npm run preview`

**Reply by juanmadev5** [reply]
Mesirve, muchas gracias!
